### DISCUSSION TITLE
Multiplatform app: where should I install a NuGet package?

### DISCUSSION THREAD

**Discussion by alessandrofrancesconi** [AvaloniaUI/Avalonia#14892] [Q&A] [answered]
Let's consider an Avalonia 11 application made of:

- Shared C# project (MyProject)
- Android target project (MyProject.Android)
- iOS target project (MyProject.iOS)

When installing a new NuGet package (e.g. "Coolpackage"), that does not come with dedicated sub-packages for each platform (e.g.: "Coolpackage.Android"), should I add it to the Shared project only, or should I add it to the Android/iOS projects also?

Thanks for your support

**Comment by thevortexcloud** [comment, marked as answer]
WIthout knowing more, install it into the shared project. You should get an implicit referernce to it IIRC. If it's only being used by a single platform then only install it for that platform, It will bloat your install size for all platforms if you don't.
